README.md

```diff
@@ -118,6 +120,20 @@ The **API endpoint** can be used to:
   scenarios; applies a cgroup/namespace isolation barrier and then
   drops privileges.
 
+## Supported platforms
+
+We continuously test Firecracker on machines with the following CPUs micro-architectures:
+- Intel Skylake
+- Intel Cascade Lake
+- AMD Zen2
+- ARM Cortex-A aarch64
+
+Firecracker is [generally available](docs/RELEASE_POLICY.md) on Intel x86_64 and AMD x86_64 CPUs that offer hardware virtualization support, and that are released starting with 2015. All production use cases should follow [these production host setup instructions](docs/prod-host-setup.md).
```

Comment:
You need to break lines at 80 chars (applies below too).

Reply:
fixed many.